specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-11-01/webapplicationfirewall.json

```diff
@@ -711,6 +749,38 @@
           "readOnly": true,
           "description": "Version of the managed rule set type."
         },
+        "ruleSetVersionState": {
```

Comment:
Should be read-only.

Reply:
This field has been removed from 20201101 version